features/upgrade/security_compliance/fips.feature

```diff
@@ -0,0 +1,122 @@
+Feature: fips enabled verification for upgrade
+  # @author xiyuan@redhat.com
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: FIPS mode checking command works for a cluster with fip mode on
+    #precondition check
+    When I run the :get admin command with:
+      | resource | mc |
+    Then the step should succeed
+    And the output should contain:
+      | 99-master-fips |
+      | 99-worker-fips |
+    
+    #check whether fips enabled for master node
+    Given I switch to the first user
+    When I run the :new_project client command with:
+      | project_name | sc-upgrade |
```

Comment:
It would be good if we confirm here that the project has been created "Then the step should succeed"